docs/design.md

```diff
@@ -98,30 +108,36 @@ By returning `EntityCommands`, you can easily chain multiple widgets together an
 Define your assets in an enum so each variant maps to a `Handle`:
```

Comment:
```suggestion
Define your assets as `const` strings that map to `Handle`s:
```
Maybe also mention that you can use an `enum` with an `Into<String>`? It might not be important enough to explain that here though.

Reply:
Updated over the last few commits